### ISSUE TITLE
how to reset preferences on cherrytreeq

### ISSUE THREAD

**Issue by J0hnnyb0y86** [giuspen/cherrytree#542]
i have installed cherrytree on my mac but i have selected tray icon on my preferences, now when i open cherrytree python run and i see the tray icon but i cant open it. can i reset to default preference?

**Comment by giuspen** [comment]
yes run cherrytree, click help--open preferences directory, close cherrytree, rename in the preferences directory config.cfg (to e.g. config.cfg.OLD), start cherrytree
